src/modules/dashboard/components/ActionsPage/ActionsComponents/DefaultAction.tsx

```diff
@@ -231,10 +231,15 @@ const DefaultAction = ({
     isSmiteAction: new Decimal(reputationChange).isNegative(),
     removedSafes,
     removedSafesString,
+    safeAddress: addedSafe && (
```

Comment:
It's a nitpick but I would call it something like `addedSafeAddress`, so it's clear what operation we are talking about. But up to you...